### ISSUE TITLE
hot chocolate idea

### ISSUE THREAD

**Issue by ryanwalexander** [simonbrowndotje/barista-express#1]
I get some amazing results by starting with around 20g of quality dark chocolate, adding some hot water from the machine, a quick 30s in the microwave, then whisking with a matcha whisk until smooth. Makes a great base on which to pour the steamed milk. Hope it helps!

**Comment by simonbrowndotje** [comment]
Thanks! I don't have a microwave, but I imagine I could get the same results by grating the chocolate first and mixing with a small amount of hot water. I received some solid cylinders of (single-origin) chocolate for Christmas, so I'll be trying this at some point soon. 🙂

**Comment by ryanwalexander** [comment]
Yes, I imagine you'd get a similar result.  It just needs a tiny bit more
heat to emulsify nicely.  Preheating the cup well with boiling water may be
enough too.  Enjoy! Merry Christmas!
